physics/box2d/Makefile

```diff
@@ -38,7 +38,7 @@ DATA		:=	data
 INCLUDES	:=	include
 GRAPHICS	:=	gfx
 GFXBUILD	:=	$(BUILD)
-# ROMFS		:=	romfs
+#ROMFS		:=	romfs
```

Comment:
Stray whitespace change

Reply:
That whitespace isn't needed and is inconsistent with the rest of the makefile. Should I leave it alone?

Reply:
I see the diff didn't show that the line immediately after that doesn't have the space. I guess yeah, you're right.